packages/splitter/splitter.stories.tsx

```diff
@@ -193,30 +200,139 @@ Hook.parameters = {
+          : SplitterPlacement.END
+    });
+    const secondSplitterBag = useSplitter({
+      type,
+      min,
+      max,
+      orientation,
+      ariaLabel,
+      defaultValueNow,
+      keyboardStep,
+      placement:
+        mode === 'rtl' && orientation === SplitterOrientation.VERTICAL
+          ? SplitterPlacement.END
+          : SplitterPlacement.START
+    });
+    const firstSeparator = firstSplitterBag.getSeparatorProps({
+      style: {
+        ...separatorStyle,
+        cursor: orientation === SplitterOrientation.HORIZONTAL ? 'row-resize' : 'col-resize'
+      }
+    });
+    const firstPrimaryPane = firstSplitterBag.getPrimaryPaneProps({
+      style: primaryPaneStyle
+    });
+
+    const secondSeparator = secondSplitterBag.getSeparatorProps({
+      style: {
+        ...separatorStyle,
+        cursor: orientation === SplitterOrientation.HORIZONTAL ? 'row-resize' : 'col-resize'
+      }
+    });
+
+    const secondPrimaryPane = secondSplitterBag.getPrimaryPaneProps({
+      style: primaryPaneStyle
+    });
+
+    return (
+      <div style={{ paddingRight: '2em', margin: '2em' }}>
+        <div style={{ padding: '2em', margin: '2em' }}>
+          <div
+            dir={mode}
+            style={{
+              ...flexContainerStyle,
+              height: '1000px',
+              flexDirection: orientation === SplitterOrientation.HORIZONTAL ? 'column' : 'row'
+            }}
+          >
+            <div
+              {...firstPrimaryPane}
+              style={{
+                ...firstPrimaryPane.style,
+                flexBasis: firstSplitterBag.valueNow
+              }}
+            >
+              Thai tabasco pepper cremini mushrooms crumbled lentils one bowl almonds delightful
+              blueberry scones simmer muffins red pepper jalapeño cherry pasta chocolate bruschetta.
+            </div>
+            <div {...firstSeparator} />
+            <div style={secondaryPaneStyle}>
+              Grains spring soba noodles pomegranate veggie burgers picnic cocoa green tea lime
+              maple orange tempeh ginger tofu leek basmati double dark chocolate figs artichoke
+              hearts raspberry fizz lemon lime minty summertime scotch bonnet pepper banana
+              four-layer pine nuts Thai sun pepper sesame soba noodles mediterranean vegetables
+              chocolate cookie. Udon noodles toasted hazelnuts peach strawberry mango ginger
+              lemongrass agave green tea homemade balsamic
+            </div>
+            <div {...secondSeparator} />
+            <div
+              {...secondPrimaryPane}
+              style={{
+                ...secondPrimaryPane.style,
+                flexBasis: secondSplitterBag.valueNow
+              }}
+            >
+              Thai tabasco pepper cremini mushrooms crumbled lentils one bowl almonds delightful
+              blueberry scones simmer muffins red pepper jalapeño cherry pasta chocolate bruschetta.
```

Comment:
Can you vary this text so we can visually observe `rtl` vs `placement` behavior applied to the story? Otherwise, it can appears as if nothing has happened.